youtube_dl_gui/locale/pt_BR/LC_MESSAGES/youtube_dl_gui.po

```diff
@@ -552,76 +522,68 @@ msgid "Password"
 msgstr "Senha"
 
 #: youtube_dl_gui/optionsframe.py:737
-#, fuzzy
 msgid "Video password"
-msgstr "SUDO senha"
+msgstr "Senha do Video"
 
 #: youtube_dl_gui/optionsframe.py:740
 msgid "Network"
-msgstr ""
+msgstr "Rede"
 
 #: youtube_dl_gui/optionsframe.py:742
 msgid "Proxy"
 msgstr "Proxy"
 
 #: youtube_dl_gui/optionsframe.py:744
-#, fuzzy
 msgid "User agent"
 msgstr "User Agent"
 
 #: youtube_dl_gui/optionsframe.py:746
 msgid "Referer"
-msgstr "Referer"
+msgstr "Refererente"
 
 #: youtube_dl_gui/optionsframe.py:749
 msgid "Logging"
-msgstr ""
+msgstr "Gerar Log"
 
 #: youtube_dl_gui/optionsframe.py:751
-#, fuzzy
 msgid "Enable log"
-msgstr "Habiliar Log"
+msgstr "Habilitar Log"
 
 #: youtube_dl_gui/optionsframe.py:752
-#, fuzzy
 msgid "View"
 msgstr "Ver Log"
 
 #: youtube_dl_gui/optionsframe.py:753
-#, fuzzy
 msgid "Clear"
 msgstr "Limpar Log"
 
 #: youtube_dl_gui/optionsframe.py:863
-#, fuzzy
 msgid "Youtube-dl command line options (e.g. --help)"
-msgstr "Argumentos de linha de comando (e.x. --help)"
+msgstr "Argumentos da linha de comando (e.x. --help)"
 
 #: youtube_dl_gui/optionsframe.py:866
-#, fuzzy
 msgid "Extra options"
-msgstr "Voltar opções"
+msgstr "Opções extras"
 
 #: youtube_dl_gui/optionsframe.py:868
 msgid "Debug youtube-dl"
-msgstr ""
+msgstr "Depurar youtube-dl"
 
 #: youtube_dl_gui/optionsframe.py:869
-#, fuzzy
 msgid "Ignore errors"
-msgstr "Ignora erros"
+msgstr "Ignorar erros"
 
 #: youtube_dl_gui/optionsframe.py:870
 msgid "Ignore youtube-dl config"
-msgstr ""
+msgstr "Ignorar configurações do Youtube-dl"
```

Comment:
Again, try to be consistent with your choices either use "Youtube-dl" in all **msgstr** fields or "youtube-dl".
For example, here you use "Youtube-dl" (capital Y) and on line 570 you use "youtube-dl".